ember-headlessui/addon/components/tabs-group/-content.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import { modifier } from 'ember-modifier';
+
+export default class Content extends Component {
+  @tracked element = null;
+  get renderContent() {
+    return this.element === this.args.selectedContent;
+  }
+  registerContent = modifier((e) => {
```

Comment:
can we not have this side-effect here? I know this pattern is common in headlessui, but it's a problematic pattern, and we don't want to propagate it.

Derived data will be more performant, have fewer layout shifts, and will in general be _way_ easier to debug.